policies/policies.go

```diff
@@ -96,14 +120,20 @@
 
 			if runMap != nil {
 				metricId := fileInfo.Name()
-				applicableMetrics[key] = append(applicableMetrics[key], metricId)
+				metrics = append(metrics, metricId)
 				runMap.MetricId = metricId
 
 				data = append(data, runMap)
 			}
 		}
+
+		// Set it and unlock
+		applicableMetrics.m[key] = metrics
+		applicableMetrics.Unlock()
+
+		log.Infof("Resource type %v has the following %v applicable metric(s): %v", key, len(applicableMetrics.m[key]), applicableMetrics.m[key])
```

Comment:
Keep in mind that the key is a concatenation of the resource types and the logging output may not look as you  would expect, e.g. 
```INFO[0002] Resource type StorageService-NetworkService-Networking-Resource has the following 4 applicable metric(s): [TransportEncryptionEnforced TLSVersion TransportEncryptionAlgorithm TransportEncryptionEnabled]  component=policies```

Reply:
yes, but that is the key that is used, so I think it is ok.